version.php

```diff
@@ -15,7 +15,7 @@
 // along with Moodle. If not, see <http://www.gnu.org/licenses/>.
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2023070300;
+$plugin->version   = 2023070300.3;
```

Comment:
We typically do not have decimal in the version. It can be the date when you made the commit.

Reply:
You are right.
We sometimes add decimal to not miss on upgrade version from upstream. But yes we can change that.

Reply:
Also, we'll bump this when we do a release regardless.